test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -362,6 +371,7 @@ public void createParticipantEmailEnabledNoVerificationWanted() {
         
         verify(accountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
         verify(account).setStatus(AccountStatus.ENABLED);
+        verify(account).setEmailVerified(Boolean.TRUE);
```

Comment:
Should createParticipantEmailEnabledVerificationWanted() also be updated?

Also, what about the case where we don't request send verify email, and we also don't provide an email?

Reply:
When we set it to unverified, we don't set the emailVerified flag at all. I thought about it... we could. But we don't really need to.

Added createParticipantWithPhoneEmailEnabledVerificationWanted() test for case of a phone being supplied where we would attempt to verify an email address.